front_end/content.py

```diff
@@ -855,7 +855,7 @@ def save_exercise_score(self, course_id, assignment_id, exercise_id, user_id, ne
 
     def get_submissions_basic(self, course_id, assignment_id, exercise_id, user_id):
         submissions = []
-        sql = '''SELECT submission_id, date, passed
+        sql = '''SELECT submission_id, date, passed, partner_id
```

Comment:
I ended up implementing this with a conditional select statement since partner_id is sometimes null and I ran into issues trying to use an INNER JOIN for this. Let me know if that is okay!